### PR TITLE
Update AOCORCF4V3 

### PR DESCRIPTION
**Pull request by 634296207**
1.Added option to compile Add the sd card or FLash select compilation function
2.Add SPL06
3.Camera control functionusing sd card or using FLash

### CODE REVIEW

File: src/main/target/AOCODARCF4V3/CMakeLists
Comment (634296207) [inline src/main/target/AOCODARCF4V3/CMakeLists:1]: Added option to compile using sd card or using FLash

File: src/main/target/AOCODARCF4V3/CMakeLists [hunk 1]
```diff
@@ -1 +1,2 @@
-target_stm32f405xg(AOCODARCF4V3)
+target_stm32f405xg(AOCODARCF4V3_SDCard)
+target_stm32f405xg(AOCODARCF4V3_Flash)
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/CMakeLists]: Added option to compile using sd card or using FLash

File: src/main/target/AOCODARCF4V3/config.c [hunk 2]
```diff
@@ -51,6 +51,7 @@ void targetConfiguration(void)
 {
     pinioBoxConfigMutable()->permanentId[0] = BOX_PERMANENT_ID_USER1;
     pinioBoxConfigMutable()->permanentId[1] = BOX_PERMANENT_ID_USER2;
+    pinioBoxConfigMutable()->permanentId[2] = BOX_PERMANENT_ID_USER3;
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/config.c:54]: Add User3 mode to control the camera

File: src/main/target/AOCODARCF4V3/target.h [hunk 3]
```diff
@@ -19,8 +19,13 @@
...
 
-#define TARGET_BOARD_IDENTIFIER         "AOF4V3"
-#define USBD_PRODUCT_STRING             "AocodaRCF4V3"
+#ifdef AOCODARCF4V3_Flash
+#define TARGET_BOARD_IDENTIFIER "AOF4V3F"
+#define USBD_PRODUCT_STRING     "AocodaRCF4V3_Flash"
+#else
+#define TARGET_BOARD_IDENTIFIER "AOF4V3SD"
+#define USBD_PRODUCT_STRING     "AOCODARCF4V3_SDCard"
+#endif
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/target.h]: Add the sd card or FLash select compilation function

File: src/main/target/AOCODARCF4V3/target.h [hunk 4]
```diff
@@ -61,6 +66,7 @@
 #define USE_BARO_BMP280
 #define USE_BARO_MS5611
 #define USE_BARO_DPS310
+#define USE_BARO_SPL06
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/target.h:64]: Add SPL06

File: src/main/target/AOCODARCF4V3/target.h [hunk 5]
```diff
@@ -136,18 +142,27 @@
...
 //******* FLASH ********
+#if defined(AOCODARCF4V3_SDCard)
+#define USE_SDCARD
+#define USE_SDCARD_SPI
+#define SDCARD_SPI_BUS          BUS_SPI3
+#define SDCARD_CS_PIN           PC0
+#define SDCARD_DETECT_INVERTED
+#define SDCARD_DETECT_PIN       PC14
+#define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
+#else
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/target.h]: Adding an sd card adapter

File: src/main/target/AOCODARCF4V3/target.h [hunk 6]
```diff
@@ -136,18 +142,27 @@
...
-
 #define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
-
+#endif
 // *************** PINIO ***************************
 #define USE_PINIO
 #define USE_PINIOBOX
 #define PINIO1_PIN                  PC5  // VTX power switcher
 #define PINIO2_PIN                  PA14  //bluetooth
+#define PINIO3_PIN                  PC15 //Camera control
```
Comment (634296207) [inline src/main/target/AOCODARCF4V3/target.h:152]: Camera control function

**Comment by 634296207** [comment]
Modify the file to fit F4v3-flash

**Comment by lida2003** [comment]
Has this PR passed self test?

**Comment by 634296207** [comment]
F405V3 Flash V3(HW) self test passed.

**Pull request merged**